kcp/pkg/deploy/assets/sample-chart/templates/webhook-config.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if not .Values.isWebhookConfigRendered }}
```

Comment:
I thought this will be dynamically configured and applied via the kyma control loop? Please correct me if I am wrong.

Reply:
Actually, this is a flag that I use to ignore this sample config file and replace it with the rendered template file for the webhook config when deploying the chart.